pkgs/by-name/nj/njq/package.nix

```diff
@@ -0,0 +1,36 @@
+{
+  lib,
+  fetchFromGitHub,
+  rustPlatform,
+}:
+rustPlatform.buildRustPackage (finalAttrs: {
+  pname = "njq";
+  version = "v0.0.3.1";
+
+  src = fetchFromGitHub {
+    owner = "Rucadi";
+    repo = "njq";
+    tag = finalAttrs.version;
+    hash = "sha256-evFKm5NYNqIIRVbtMlGFY0k+0sJnkTCTIbTH7w9jQtk=";
+  };
+
+  postPatch = ''
+    cp -L ${./Cargo.lock} Cargo.lock
+  '';
+
+  cargoLock = {
+    lockFile = ./Cargo.lock;
+    outputHashes = {
+      "snix-eval-0.1.0" = "sha256-ciBpaYvwcjJ1YN2dc6xyQVebsERIKZ+p2q+xs6sO+CE=";
+    };
+  };
```

Comment:
please, use the cargo.lock included in the upstream repo

Reply:
Yeah. I understand. We just prefer the in repo ones because we are trying to remove bloated and disk-heavy lock files from Nixpkgs.